editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -698,6 +699,297 @@ describe('Absolute Resize Strategy', () => {
       projectDoesHonourSizeProperties(340, 275),
     )
   })
+  it('resizes a component instance that honours the size properties by updating the instance', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      ComponentsHonouringPropsStylesProject,
+      'await-first-dom-report',
+    )
+
+    const target = EP.fromString('storyboard/scene/mycompdiv')
+
+    await renderResult.dispatch([selectComponents([target], false)], true)
+
+    expect(renderResult.renderedDOM.getByTestId('parent-resize-label')).toBeTruthy()
+
+    const dragDelta = windowPoint({ x: 0, y: 30 })
+    await resizeElement(renderResult, dragDelta, EdgePositionBottom, emptyModifiers)
+    await renderResult.getDispatchFollowUpActionsFinished()
+    expect(getPrintedUiJsCode(renderResult.getEditorState()))
+      .toEqual(`import * as React from 'react'
+import { Scene, Storyboard } from 'utopia-api'
+
+export const MyCompDiv = (props) => {
+  return (
+    <div
+      data-uid='mycompdivinnerdiv'
+      data-testid='mycompdivinnerdiv'
+      style={props.style}
+    >
+      MyCompDiv
+    </div>
+  )
+}
+
+export const MyCompFrag = (props) => {
+  return (
+    <>
+      <div
+        data-uid='mycompfraginnerdiv'
+        data-testid='mycompfraginnerdiv'
+        style={props.style}
+      >
+        MyCompFrag
+      </div>
+    </>
+  )
+}
+
+export var storyboard = (
+  <Storyboard data-uid='storyboard'>
+    <div
+      data-uid='scene'
+      style={{
+        backgroundColor: '#aaaaaa33',
+        position: 'absolute',
+        left: 118,
+        top: 172,
+        width: 300,
+        height: 300,
+      }}
+    >
+      <MyCompDiv
+        data-uid='mycompdiv'
+        style={{
+          position: 'absolute',
+          width: 137,
+          height: 121,
+          left: 137,
+          top: 36,
```

Comment:
I wish it would be more clear that these are the four lines out of 70 that is interesting